homeassistant/auth/providers/command_line.py

```diff
@@ -126,7 +126,6 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         meta = self._user_meta.get(credentials.data["username"], {})
         return UserMeta(
             name=meta.get("name"),
```

Comment:
Why would command_line override the name on each login ? I feel like this is being built for your specific use case.

Reply:
Because the command line auth provider retrieves its information from an external data source like a single signon database, and that's where user information should reside when this auth method is chosen.